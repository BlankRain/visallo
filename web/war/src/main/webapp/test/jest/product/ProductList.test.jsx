import React from 'react'
import renderer from 'react-test-renderer'
import ProductList from 'product/ProductList'

it('should have a product list', () => {
    const props = {
        products: [],
        onCreate: () => {},
        onLoadProducts: () => {},
        types: [],
        registry: {},
        workspace: { editable: false },
        user: { privileges: [] },
        status: { loading: false, loaded: false }
    }
    const component = renderer.create(<ProductList {...props} />, {})
    expect(component.toJSON()).toMatchSnapshot()
})
