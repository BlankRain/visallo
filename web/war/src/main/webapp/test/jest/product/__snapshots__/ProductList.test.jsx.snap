exports[`test should have a product list 1`] = `null`;
